### DISCUSSION TITLE
Conditional config

### DISCUSSION THREAD

**Discussion by klaushipp** [kovidgoyal/kitty#3280] [Q&A] [answered]
Hi,
first of all, awesome project. Thank you for sharing it with us.
I'm using the config file of kitty on two different systems, syncing it with the help git.
I would like to have almost the same configuration for both system, but with small deviations (font size for example).
Is there a way of insert if-conditions or something like that in the config file to provide such feature?
Thank you in advance. I will continue searching for it on the docs in the meantime.

**Comment by Luflosi** [comment, marked as answer]
See https://github.com/kovidgoyal/kitty/issues/811#issuecomment-414186903.

**Reply by klaushipp** [reply]
Thank you! Perfect!
